### PR TITLE
[fix] Use local instances of expandOnLoad

### PR DESCRIPTION
**Pull request by billyhunt**
Updated version of https://github.com/CenterForOpenScience/osf.io/pull/5233
Related to https://github.com/CenterForOpenScience/osf.io/pull/4958/files

When adding contributors, the top level is not expanding on staging.  This seems to be correlated to a move of multiple expandOnLoad functions to a global helper.  We are going to go back to using local instances for now.

### CODE REVIEW

**Comment by sloria** [comment]
jshint is complaining; looks like a missing import maybe?

**Comment by icereval** [comment]
@caneruguz fixed a handful of similar issues in0.66.0-develop

File: website/static/js/nodeSelectTreebeard.js [hunk 1]
```diff
@@ -78,7 +133,7 @@ function NodeSelectTreebeard(divID, data, nodesState) {
         }
     });
     var grid = new Treebeard(tbOptions);
-    projectSettingsTreebeardBase.expandOnLoad.call(grid.tbController);
+    expandOnLoad.call(grid.tbController);
```
Comment (caneruguz) [inline website/static/js/nodeSelectTreebeard.js]: This call can be made with treebeard's existing hook : onload, unless that's already defined. The manual changes we made to 0.66 were like this. https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/nodesPrivacySettingsTreebeard.js#L71

File: website/static/js/nodeSelectTreebeard.js [hunk 2]
```diff
@@ -78,7 +133,7 @@ function NodeSelectTreebeard(divID, data, nodesState) {
         }
     });
     var grid = new Treebeard(tbOptions);
-    projectSettingsTreebeardBase.expandOnLoad.call(grid.tbController);
```
Comment (caneruguz) [inline website/static/js/nodeSelectTreebeard.js]: What's the problem with using expandOnload from a central file?

**Comment by caneruguz** [comment]
@billyhunt sorry we had to fix some conflicts manually for 0.66 that put this code into conflict. Treebeard controller changes so we removed tbController. Let me know if you need any help moving this along.

**Comment by caneruguz** [comment]
This can be closed in favor of this: https://github.com/CenterForOpenScience/osf.io/pull/5429

**Pull request closed without merging**